src/test/java/org/inferred/freebuilder/processor/property/BiMapMutateMethodTest.java

```diff
@@ -233,6 +233,24 @@ public void callSetValueOnEntryChecksArguments() {
         .runTest();
   }
 
+  @Test
+  public void callSetValueOnEntryChecksDuplicateValue() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("value already present: ");
```

Comment:
I'm not sure for your framework the cleanest way to change a literal "beta" -> beta. I could do `values.example(1).replace('\"', '')` but I was thinking there was a better way.

Reply:
You'd have to add that method to ElementFactory. `exampleToString`?